doc pass

jawa9000 opened the pull request:
Fixed a few typos (IOS vs iOS) and word-smithed a few summaries and
descriptions.

jawa9000 committed:
doc pass

Fixed a few typos (IOS vs iOS) and word-smithed a few summaries and
descriptions.

hansemannn commented:
Hi @jawa9000, before we can merge your PR, you have to do three more things:
- Create a [JIRA ticket](jira.appcelerator.org) to link this issue to
- Update your fork to the latest master to avoid merge conflicts
- Sign the CLA agreement to open source your work.

If you have any further question, let us know. Thank you!

jawa9000 committed:
updated Window.yml per TIDOC-2375

Added documentation to the Titanium.UI.Window API page to include the
new 5.0.0 feature that could prevent accidentally app closure from
hitting the back button too many times.

jawa9000 committed:
Merge remote-tracking branch 'appcelerator/master'

jawa9000 committed:
Merge remote-tracking branch 'appcelerator/master'

hansemannn commented:
Hi @jawa9000, the PR still has merge conflicts, the wrong file names and a missing CLA that needs to be signed. Please update your PR so we can merge it, thanks!

jawa9000 committed:
renamed TableView.iml to TableView.yml

Correcting a typo I made.

jawa9000 committed:
Merge remote-tracking branch 'appcelerator/master'

# Conflicts:
#	apidoc/Titanium/UI/Window.yml

hansemannn commented:
Closing in favor of https://github.com/appcelerator/titanium_mobile/pull/7646 to resolve the merge problems in Window.yml.
